README.md

```diff
@@ -171,6 +171,12 @@ optional arguments:
                         defaults to `on` for the `json` format. This flag has
                         no effect on the `cyclonedx-json` or `cyclonedx-xml`
                         formats. (default: auto)
+  --aliases [{on,off,auto}]
+                        includes alias IDs for each vulnerability; `auto`
+                        defaults to `off` for the `column` and `markdown`
+                        formats. This flag has no effect on the
+                        `cyclonedx-json`, `cyclonedx-xml`, and `json`
```

Comment:
What's the reason for forcing this on for the JSON format? It's not a big deal since the alias list won't be large but it's not consistent with the `--desc` flag.

Also if I'm reading this right, the `off` option works identical to `auto` since the JSON format always includes aliases.

I think we should either:
- Allow aliases to be toggled for the JSON format, `auto` defaults to `on` for JSON (this makes the most sense to me).
- Remove the `auto` value.

Thoughts @woodruffw @di?

Reply:
Yeah, agreed -- I think forcing on (rather than having the default be on) will cause user confusion. I'll make a suggested change below.

Reply:
My initial thoughts were that forcing it to be included would be inobtrusive enough that it wouldn't warrant a toggle (as with the case for descriptions).

@tetsuo-cpp the difference between what I previously implemented as `off` and `auto` was that passing `pip-audit ... -f json --aliases off` as a flag would not affect the JSON output, and aliases would have always been included. With `pip-audit ... -f json --desc off`, descriptions would be omitted from the JSON output.

That being said, I agree for the sake of consistency that the behaviour should match `--desc`, and as such have added the necessary code to support that.